plugins/inputs/passenger/passenger.go

```diff
@@ -0,0 +1,217 @@
+package passenger
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/influxdb/telegraf/plugins/inputs"
+	"golang.org/x/net/html/charset"
+)
+
+type passenger struct {
+	Command string
+}
+
+type info struct {
+	Passenger_version  string `xml:"passenger_version"`
+	Process_count      int    `xml:"process_count"`
+	Capacity_used      int    `xml:"capacity_used"`
+	Get_wait_list_size int    `xml:"get_wait_list_size"`
+	Max                int    `xml:"max"`
+	Supergroups        struct {
+		Supergroup []struct {
+			Name               string `xml:"name"`
+			Get_wait_list_size int    `xml:"get_wait_list_size"`
+			Capacity_used      int    `xml:"capacity_used"`
+			Group              []struct {
```

Comment:
Group also has a `name` and `app_root` parameter, these seem like they would be useful tags